src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,145 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static int CAPACITY = 10;
```

Comment:
Shouldnt be static
```suggestion
    private static int CAPACITY = 10;
```

Reply:
why it Shouldn't be static?

Reply:
static applies to all instances of class.
So if you change static in one class, it will affect others.

Make it or constant, e.g. private static final or plain field depends on context

Reply:
ok i'll mark it. thx